NOTICES

```diff
@@ -0,0 +1,33 @@
+# Notices
+
+## Code of Conduct
+
+Contact for Code of Conduct issues or inquires: legal@linuxfoundation.org
+
+## Exclusions
+
+- For FDC3 1.0, please see [Patents-FDC3-1.0.md](Patents-FDC3-1.0.md)
+
+This section includes any Exclusion Notices made against a Draft Deliverable or Approved Deliverable as set forth in the Community Specification Development License.  Each Exclusion Notice must include the following information:
+
```

Comment:
We've removed a section here since the procedure for signing up doesn't match what we actually do.

Can someone Legal review?

Reply:
https://github.com/finos/FDC3/blob/master/README.md#how-do-you-become-a-participant-editor-or-maintainer

Reply:
@mindthegab can you add a legal reviewer?

Reply:
Currently, people send an email to fdc3-participants+subscribe@finos.org

containing something like:

> HI, my name is <FirstName LastName> and I'd like to formally participate to the FDC3 standard process. I plan to contribute as <individual|on behalf of organizationName> and I have reviewed the policies described at https://github.com/finos/community/tree/master/governance/Standards-Projects . Thank you!